### ISSUE TITLE
Suggestion: Void Cell / Void Card

### ISSUE THREAD

**Issue by eslachance** [ExtraCells/ExtraCells2#292]
Since my suggestion was summarily dismissed by @yueh on AppliedEnergistics/Applied-Energistics-2#1875 , perhaps I'd have a little more chance here as an add-on (since I use EC2). Here is the suggestion as posed there (but additionally, this could be a special "Void Cell" instead of a card...)

At the moment there is one thing that prevents a 100%, erm, "easy" setup in AE2: Voiding stuff when you have too much of it. Aka, keep only X number of stacks of cobble and delete the rest. While it is possible to send items to a trashcan or other voiding apparatus with an export bus and a level emitter, having one such setup for each item is a little complex.

My suggestion would be a Void Card, which would act similarly to the Void Upgrade from both JABBA and Storage Drawers. When added to an ME Drive or an ME Chest, anything that cannot fit within the storage cells would be voided. A single ME chest with a partitioned cell and this card would then be suitable to replace about a dozen barrels/drawers with void upgrades (with storage buses), for the things you know you'll get a lot of because of automation, farms, and quarries.

**Comment by ruifung** [comment]
I do not see the point in this. It is trivial to do using a matter condenser, storage bus and properly setting the priorities.

It's also somewhat difficult to implement for cells as they report a fixed size, and different behavior between the reported free bytes and the behavior while injecting would probably be a bad idea.

tl;dr: No. You don't need export bus / level emitter for this - just use priorities.
